workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -111,6 +115,16 @@ public RedirectedProcess AddErrorProcessing(Action<string> errorProcessor)
             return this;
         }
 
+        /// <summary>
+        ///     Adds accumulated output processing
```

Comment:
```suggestion
        ///     Adds the specified action to a list of processors. These actions are called at the end of running the specified process and passes in as an argument the complete log of that process.
```

Reply:
Let me update other processors as well along to add more clarity